version/1/4/bs/code_of_conduct.md

```diff
@@ -0,0 +1,45 @@
+# Saradnik Pakt: Kodeks Ponašanja
+
+## Naš zalog
+
+U interesu ohrabrujući otvoren i dobrodošlice okruženje,mi kao saradnici i održavatelji obećamo da za sviju ko učestvuju u našem projektu i zajednicu, njihovo iskustvo  će biti bez uznemiravanje, bez obzira na starost, veličinu, invalidnost, nacionalnost, rodnog identiteta i izraz, nivo iskustva, državljanstvo, izgled, porod, vjera, ili seksualni identitet i orijentacija.
```

Comment:
This is also Croatian but this is my attempt at rewritting that passage

U interesu ohrabrivanja otvorenog i pristupačnog okruženja, mi kao saradnici i odrzavatelji obecavamo svima koji ucestvuju u nasem projektu i zajednici da ce njihovo iskustvo biti bez uznemiravanja, bez diskriminacije na osnovu dobi, invalidnosti, nacionalnosti, rodnog identiteta i izraza, nivoa iskustva, drzavljanstva, izgleda, poroda, vjere ili sekusalnog identiteta ili orijentacije.